README.md

```diff
@@ -206,15 +209,15 @@ Let's trying running it.
 
 ### :keyboard: Activity: Run your image
 
-1. Find your image information by typing `Docker image ls`.
-   ![screenshot of output from Docker image ls command: lists docker images, REPOSITORY TAG and docker URL](https://i.imgur.com/UAwRXiq.png)
+1. Find your image information by typing `docker image ls`.
+   ![screenshot of output from Docker image ls command: lists docker images, REPOSITORY TAG and docker URL](https://i.imgur.com/UAwRXiq.png)<!-- This screenshot should be changed. -->
 1. Use the following command to run a container from your image:
    ```
    docker run -d -it --rm -p 8080:80 --name ttt <YOUR_IMAGE_NAME:TAG>
    ```
 1. Replace `YOUR_IMAGE_NAME` with your image name under the `REPOSITORY` column.
 1. Replace `TAG` with the image tag under the `TAG` column
-   ![example of running the docker command listed above](https://i.imgur.com/hr6N9nk.png)
+   ![example of running the docker command listed above](https://i.imgur.com/hr6N9nk.png)<!-- This screenshot should be changed. -->
```

Comment:
```suggestion
   ![example of running the docker command listed above](https://user-images.githubusercontent.com/3250463/219255534-f11b20e8-65de-4f4a-a033-f312ddf507fb.png)
```

---

## Image:
<img width="586" alt="Screen Shot 2023-02-15 at 6 27 01 PM" src="https://user-images.githubusercontent.com/3250463/219255534-f11b20e8-65de-4f4a-a033-f312ddf507fb.png">

Reply:
This screenshot does not agree with the instruction.
How would you like the interactive and tty options `-it` and the port option `-p 8080:80`?

Reply:
Oh ya, I forgot I tested removing those arguments. My assumption was that those arguments were for the old docker image. They're not needed for this new one. So the example on line 213/216 would be changed to match this screenshot. **IF** my assumption is right.

Reply:
Well, the option `-p 8080:80` is actually needed if learners would like to open the page (index.html).
I have opended another PR #22.